sgkit/stats/aggregation.py

```diff
@@ -0,0 +1,53 @@
+import xarray as xr
+from xarray import DataArray, Dataset
+
+
+def allele_count(ds: Dataset) -> DataArray:
```

Comment:
I think this should be called `count_alleles`. Functions are normally verbs.

Reply:
https://github.com/pystatgen/sgkit/pull/36/commits/802c8d72dd1289ff4901db24755ff44ca51ec715#diff-e2edcf5f78ccf0e17093a6cbe4afeda6R5